lib/src/base-styles/_nx-page.scss

```diff
@@ -29,19 +31,30 @@
   }
 }
 
+.nx-page-title__sub-title {
+  @include nx-truncate-ellipsis;
+  border-left: 1px solid $nx-indigo-100;
+  margin: 0;
+  padding-left: $nx-spacing-md;
+}
+
+.nx-page-title__description {
+  @include container-vertical;
+  grid-area: description;
+  margin-top: $nx-spacing-xs;
+
+  & + .nx-page-title__tags {
```

Comment:
The design requires that the first block below the page title have `8px` of top spacing, and the next one should have `16px`. The description always precedes the tags so give both `8px` in case they are alone, and give the tags `16px` only when they follow the description.